ios/RNWebrtcVad.m

```diff
@@ -33,53 +34,86 @@ @implementation RNWebrtcVad {
     self.audioData = nil;
```

Comment:
i don't know if this is valid, but could you just call the `dealloc` method here instead of duping the `nil`ing?

Reply:
Good question. Because we’re have ARC (automatic reference counting) enable on our compiler settings, once there are no more references the ru time automatically GC’s it which falls dealloc. Actually trying to manually dealloc will cause a compiler error.